File: tienda/models/order.go
package models

import "time"

type Order struct {
	ID       string    `bson:"_id,omitempty"`
	Name     string    `bson:"name"`
	Address  string    `bson:"address"`
	Products []string  `bson:"products"`
	Date     time.Time `bson:"date"`
}

File: backend/models/Order.js
import mongoose from "mongoose";

const orderSchema = new mongoose.Schema({
    name: { type: String, required: true },
    address: { type: String, required: true },
    products: [{
        productId: { type: mongoose.Schema.Types.ObjectId, ref: 'Product' },
        quantity: { type: Number, required: true }
    }],
    date: { type: Date, default: Date.now },
});

export default mongoose.model("Order", orderSchema);